internal/dinosaur/pkg/gitops/default_central_test.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 func TestDefaultCentral(t *testing.T) {
-	p := getDummyCentralParams()
+	p := getDummyCentralParams(false)
```

Comment:
would it be possible to just ad-hoc set isInternal to true when needed for testing ? 
```suggestion
	p := getDummyCentralParams()
	p.IsInternal = false / true
```